test/test.js

```diff
@@ -23,38 +23,40 @@ var cleaner = function (cb) {
     fs.rmdir(path.resolve(__dirname, 'report'), function (err) {
       //ignore errors
       cb();
-    })
+    });
   });
 };
 describe('nemo-screenshot', function () {
+
   before(function (done) {
     cleaner(function () {
-      nemo = Nemo(basedir, config, function (err) {
+      Nemo(basedir, config, function (err, _nemo) {
         if (err) {
           done(err);
         }
+        nemo = _nemo;
```

Comment:
it does not get set to scoping issues.

I tried to run the tests with 

```
nemo = nemo
```

and got this as the output

```
LM-NYC-00875272:nemo-screenshot gacsapo$ npm test

> nemo-screenshot@2.0.0 test /Users/gacsapo/Documents/nemo/nemo-screenshot
> grunt

Running "simplemocha:all" (simplemocha) task


  nemo-screenshot
    1) will get @setup@
    2) will use @snap@ to take a screenshot
    3) will use @done@ to take a screenshot
    4) will use @done@error@ to take a screenshot in an error scenario
    5) "after all" hook


  0 passing (1s)
  5 failing

  1) nemo-screenshot will get @setup@:
     TypeError: Cannot read property 'screenshot' of undefined
    at Context.<anonymous> (test/test.js:56:16)
  

  2) nemo-screenshot will use @snap@ to take a screenshot:
     TypeError: Cannot read property 'driver' of undefined
    at Context.<anonymous> (test/test.js:61:9)
  

  3) nemo-screenshot will use @done@ to take a screenshot:
     TypeError: Cannot read property 'screenshot' of undefined
    at Context.<anonymous> (test/test.js:71:9)
  

  4) nemo-screenshot will use @done@error@ to take a screenshot in an error scenario:
     TypeError: Cannot read property 'screenshot' of undefined
    at Context.<anonymous> (test/test.js:78:9)
  

  5) nemo-screenshot "after all" hook:
     TypeError: Cannot read property 'driver' of undefined
    at Context.<anonymous> (test/test.js:44:11)
```